TARGETS

```diff
@@ -73,7 +73,7 @@ sanitizer = read_config("fbcode", "sanitizer")
 
 # Let RocksDB aware of jemalloc existence.
 # Do not enable it if sanitizer presents.
-if default_allocator.startswith("jemalloc") and sanitizer == "":
+if is_opt_mode and default_allocator.startswith("jemalloc") and sanitizer == "":
```

Comment:
QQ: Why is it a problem only in opt mode? Shouldn't it fail similarly in dev mode? is default_allocator not set in dev mode?

Reply:
Its a problem in dev mode. The opt mode is fine, but the dev mode doesn't seem to link with jemalloc.

Reply:
Sorry I mean the inverse: why is it a problem only in dev mode?
(Had an offline chat with @anand1976 , and we don't know the real root-cause yet).

Reply:
Found out that in dev mode ASAN is enabled and ASAN has its own allocator.